client/sites-dashboard/components/link-in-bio-banner/link-in-bio-banner.tsx

```diff
@@ -0,0 +1,62 @@
+//todo use variables where possible and check fontSize use in code base to make sure consistent
+import styled from '@emotion/styled';
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { close } from '@wordpress/icons';
+import { ComponentProps } from 'react';
+import { useLinkInBioBanner } from './use-link-in-bio-banner';
+
+export const Root = styled( 'div' )( {
+	position: 'relative',
+	display: 'flex',
+	backgroundColor: '#D0CCE3',
+	borderRadius: '3px',
+} );
+
+export const Title = styled( 'h3' )( {
+	fontWeight: 500,
+	fontSize: 20,
+	color: '#000000',
+	margin: 0,
+} );
+
+Title.defaultProps = { children: __( 'Your digital identity' ) };
+
+export const Description = styled( 'p' )( {
+	margin: 0,
+} );
+
+Description.defaultProps = {
+	children: __( 'Show the world what you have to offer with a Link to Bio site.' ),
+};
```

Comment:
I haven't seen this way of setting the text in a styled paragraph before! Convenient!
This uses the `__` function imported directly via `@wordpress/i18n`, but when we're dealing with React we usually use the `__` function that's returned by `@wordpress/react-i18n`'s `useI18n()` hook. In theory the React version means the component is able to re-render if the language ever changed. Although we never actually do that in practice—changing language always request a page refresh in Calypso.

Reply:
Ok, will switch over to the hook.

Reply:
After posting, I realised it wouldn't be possible to use the hook in this object. I've opted to keep it as-is and depend on the full refresh. Let me know if that's going to be problematic, and I can switch to a component.

Reply:
Yeah it'd be a bit of a change having to add a wrapper to the component just so you can use the hook. I think it's fine to leave as is, I was just thinking out loud that I hadn't seen it done this way before with default `children` props 😅